internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/file.go

```diff
@@ -61,8 +61,9 @@ func (b *Bridge) setupOnFileOpen() {
 
 			// Update file entity
 			file.Mapping = sink.Table.Mapping
-			file.StagingStorage.Provider = stagingFileProvider // staging file is provided by the Keboola
-			file.TargetStorage.Provider = targetProvider       // destination is a Keboola table
+			file.StagingStorage.Provider = stagingFileProvider      // staging file is provided by the Keboola
+			file.TargetStorage.Provider = targetProvider            // destination is a Keboola table
+			file.TargetStorage.Import.SinkLimit = b.config.JobLimit // specific sink condition for keboola provider
```

Comment:
This is moment, when bridge `jobLimit` is injected into file `sinkLimit`. For another provider it would be done in similar way, but it will be new configuration name.